netlib/websockets/frame.py

```diff
@@ -128,17 +141,16 @@ def __bytes__(self):
             # '!Q' = pack as 64 bit unsigned long long
             # add 8 bytes extended payload length
             b += struct.pack('!Q', self.payload_length)
+        else:
+            raise ValueError("Payload length exceeds 64bit integer")
```

Comment:
Scratch my previous comment here. `bytes(invalid_frameheader)` should (as you said) raise a ValueError. Did not consider that it was called that way. 
My reasoning was that `foo.bar()` should never raise one, as we don't have any arguments here. This does not apply here, as the object itself is an argument to the `bytes()` builtin.